esp32-led-blink-sdk/main/BridgingHeader.h

```diff
@@ -0,0 +1,17 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors.
```

Comment:
Could you make sure to use 2024 as the copyright date on these new additions

Reply:
changed in all files of the project